src/phantom.js

```diff
@@ -94,6 +95,12 @@ export default class Phantom {
         this.process.stdout.pipe(new Linerstream()).on('data', data => {
             const message = data.toString('utf8');
             if (message[0] === '>') {
+                //Server end has finished NOOP,
+                //lets allow NOOP again..
+                if (message === '>NOOP') {
+                    this.doing_NOOP = false;
+                    return;
+                }
```

Comment:
Move this after `const json = message.substr(1)` and do `if(message === 'NOOP')` instead.